src/index.js

```diff
@@ -27,7 +28,11 @@ const generateMutationQuery = require('./generate-mutation-query');
 		core.debug(JSON.stringify(resource));
 
 		// A list of columns that line up with the user entered project and column
-		const mutationQueries = generateMutationQuery(resource, project, column, nodeId);
+		const mutationQueries = generateMutationQuery(resource, project, column, nodeId, action);
+		if ((action === 'delete' || action === 'archive') && mutationQueries < 1) {
```

Comment:
This should be `mutationQueries.length < 1`

Reply:
@ksvladimir Thanks! Fixed it